### PR TITLE
Upgrade Python tutorial to Python 3 maybe?

### PR DESCRIPTION
**Pull request by shaform**
I found that the assignments start to use Python 3 syntax by default. Moreover, Python 2 is not officially supported in the assignment 2. Maybe then it would be a good idea to update Python version in the tutorial as well to avoid confusions?

### CODE REVIEW

File: python-numpy-tutorial.md [hunk 1]
```diff
@@ -108,66 +108,66 @@ familiar from other programming languages.
 
 ```python
 x = 3
-print type(x) # Prints "<type 'int'>"
-print x       # Prints "3"
-print x + 1   # Addition; prints "4"
-print x - 1   # Subtraction; prints "2"
-print x * 2   # Multiplication; prints "6"
-print x ** 2  # Exponentiation; prints "9"
+print(type(x)) # Prints "<type 'int'>"
```
Comment (omoindrot) [inline python-numpy-tutorial.md]: In python 3.5, I have:
```python
# Prints "<class 'int'>"
```

File: python-numpy-tutorial.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ Like lists and dictionaries, we can easily construct sets using set comprehensio
 ```python
 from math import sqrt
 nums = {int(sqrt(x)) for x in range(30)}
-print nums  # Prints "set([0, 1, 2, 3, 4, 5])"
+print(nums)  # Prints "set([0, 1, 2, 3, 4, 5])"
```
Comment (omoindrot) [inline python-numpy-tutorial.md]: I get:
```python
# Prints "{0, 1, 2, 3, 4, 5}"
```

**Comment by omoindrot** [comment]
Thanks you so much for the pull request, you're right it's time to change to python 3 !

**Comment by shaform** [comment]
Hi @omoindrot,
Thanks for the review, I've fixed the points you mentioned.
Just curious if you are using some automatic tools to check python code in a markdown document. (Because I actually made this PR manually, which was a little bit tedious 😂.)

**Comment by omoindrot** [comment]
Haha nothing automatic 😄 
Could you fix all the `<class ...>` instead of `<type ...>` please?

**Comment by shaform** [comment]
All fixed~

**Pull request merged**